libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -96,6 +104,27 @@ export class PickingComponent extends PickingListMixin(LitElement) {
     this.items[productIndex].status = ItemsFilters.NotPicked;
 
     this.items = [...this.items];
+
+    this.tabRef.value?.dispatchEvent(
+      new CustomEvent('click', {
+        composed: true,
+        bubbles: true,
+      })
+    );
+
+    setTimeout(() => {
+      window.scrollTo({
+        left: 0,
+        top: this.productCardRef.value?.offsetTop,
+        behavior: 'smooth',
+      });
+
+      (
+        this.productCardRef.value?.renderRoot.querySelector(
+          'oryx-cart-quantity-input'
+        ) as HTMLElement
+      )?.focus();
+    }, 100);
```

Comment:
I wonder if there is a better way of scheduling scrolling without setTimeout with 100ms delay as it seems like an unreliable way of waiting for a re-render.
What about trying to use `this.whenReady()` promise instead? In theory this should wait until the tabs are rendered and then you could scroll to it.

Reply:
Don't really know if we can inject that directly into the component :(